src/gsy_e/models/area/__init__.py

```diff
@@ -130,6 +140,7 @@ def __init__(self, name: str = None, children: List["Area"] = None,
             raise AreaException("A leaf area can not have children.")
         self.strategy = strategy
         self._config = config
+        event_list = event_list if event_list is not None else []
```

Comment:
Nit
```suggestion
        event_list = event_list or []
```

Reply:
This is right, and I feel that it would minimize the chances of us passing ANY non None object and it ending up being assigned to the event list, so imo it makes sense to adapt the proposal

Reply:
If the user passes a non-expected None object, I think it's good to let the execution crash. Also, `if param is not None else []` is just the safe equivalent of `= []`, which is what I wanted to fix. If that's ok for you, I would keep this as it is.